tests/monitors/dotnet/dotnet_test.py

```diff
@@ -0,0 +1,56 @@
+from functools import partial as p
+from textwrap import dedent
+import pytest
+import sys
+
+from helpers.util import wait_for, run_agent
+from helpers.assertions import *
+
+pytestmark = [
+    pytest.mark.skipif(sys.platform != "win32", reason="only runs on windows"),
+    pytest.mark.windows,
+    pytest.mark.dotnet,
+]
+
+
+def test_dotnet():
+    config = dedent(
+        """
+        monitors:
+         - type: dotnet
```

Comment:
Does this just pull metrics for every .NET process on Windows or what?

Reply:
yep it's exactly what our perfcounter monitor does now